webpack.config.js

```diff
@@ -40,7 +40,7 @@ module.exports = {
   },
   output: {
     filename: 'moment-range.js',
-    library: 'moment-range',
+    library: 'MomentRange',
```

Comment:
Don't you specify file names for the import (ES6) or require (commonJS). In that case, I think the changed name should not effect these cases?
I could not find any option in the webpack configuration to specify the name for the browser case.